rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -341,6 +341,11 @@ void debugger_frame::UpdateUnitList()
 		idm::select<SPUThread>(on_select);
 	}
 
+	// Set thread to main_thread by default.
+	if (m_choice_units->count() > 1)
```

Comment:
at this location this will reset the thread to 1 on every single change of thread counts